test/datahike/test/migrate_test.clj

```diff
@@ -226,3 +219,26 @@
                                :cache-size         0
                                :index :datahike.index/persistent-set
                                :attribute-refs?    false}))
+
+(defn- all-true? [c] (every? true? c))
```

Comment:
Might make sense to move `all-true?` and `all-eq?` to the `datahike.test.utils` namespace at some point, so we could them also in other tests.

Reply:
Thought of that but held back. Since you mention it though, why not now? So that they're more visible.